WooCommerce/Classes/ViewRelated/Orders/Order Filters/FilterOrderListViewModel.swift

```diff
@@ -210,5 +232,30 @@ private extension FilterOrderListViewModel.OrderListFilter {
         static let rowTitleProduct = NSLocalizedString("filterOrderListViewModel.OrderListFilter.rowTitleProduct",
                                                        value: "Product",
                                                        comment: "Row title for filtering orders by Product.")
+        static let rowCustomer = NSLocalizedString("filterOrderListViewModel.OrderListFilter.rowCustomer",
+                                                   value: "Customer",
+                                                   comment: "Row title for filtering orders by customer.")
+    }
+}
+
+extension CustomerFilter: FilterType {
+    /// The user-facing description of the filter value.
+    var description: String {
+        let fullName = ((firstName ?? "") + " " + (lastName ?? "")).trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
Nit: this can be simplified:

```suggestion
        let fullName = [firstName, lastName]
            .compactMap { $0 }
            .joined(separator: " ")
            .trimmingCharacters(in: .whitespacesAndNewlines)
```